rails/app/views/search/get_current_material_unassigned_collections.html.haml

```diff
@@ -0,0 +1,91 @@
+.search_materials
+  - if @material.length == 1
+    - material = @material[0]
+    - if material.is_a? ::Investigation
+      - investigation_name = material.name
+      - activity_name = ""
+      - external_activity_name = ""
+      - existing_collections = nil
+    - elsif material.is_a? ::Activity
+      - investigation_name = ""
+      - external_activity_name = ""
+      - parent_name = (material.parent) ? material.parent.name : ""
+      - activity_name = material.name
+      - existing_collections = []
+      - if material.parent
+        - existing_collections = MaterialsCollection.joins(:materials_collection_items).where(:materials_collection_items => {:material_id=>material.parent.id,:material_type=>"Investigation"})
+    - elsif material.is_a? ::ExternalActivity
+      - investigation_name = ""
+      - activity_name = ""
+      - external_activity_name = material.name
+      - existing_collections = MaterialsCollection.joins(:materials_collection_items).where(:materials_collection_items => {:material_id=>material.id,:material_type=>"ExternalActivity"})
+  - else
+    - investigation_name = ""
+    - parent_name = (@material[0].parent) ? @material[0].parent.name : ""
+    - activity_name = @material.collect{|item| item.name}.join('<br/>')
+
+  - activity_name = activity_name || ""
+  - external_activity_name = external_activity_name || ""
+  - if investigation_name.length > 0
+    %div
+      %table.material_detail
+        %tr
+          %td
+            = investigation_name
+  - if activity_name.length > 0
+    %table.material_detail
+      %tr
+        %td
+          = activity_name.html_safe
+      %tr
+        %td.title Part&nbsp;Of:
+        %td
+          = parent_name
+  - if external_activity_name.length > 0
+    %div
+      %table.material_detail
+        %tr
+          %td
+            = external_activity_name
```

Comment:
With the new assumption that this is always an external activity, and if this file is moved into the external_activities folder, the code above can be simplified to remove the conditional logic.